apps/ffdweb-site/src/app/(homepage)/page.tsx

```diff
@@ -98,7 +100,13 @@ export default async function Home() {
           'Learn how decentralized technologies are creating a better future for a better web. Explore educational content, research papers, tutorials, interactive content, and more resources from FFDW and beyond.',
         ]}
       >
-        <div className="bg-brand-primary-800 grid h-96 w-full grid-cols-3 gap-4" />
+        {/* <CardGrid cols="mdThree">
```

Comment:
Commented out as we [don't want to show cards for now. ](https://filecoinfoundation.slack.com/archives/C06PL3W4FPE/p1741865993744089?thread_ts=1741864069.939209&cid=C06PL3W4FPE)